### PR TITLE
Update to latest versions

### PR DESCRIPTION
**Pull request by lesv**
1. change the version properties to match the artifactId’s.
2. update to latest versions
3. Do the same for check style config, but don’t release — not
significant t changes.
4.

### CODE REVIEW

**Comment by lesv** [comment]
Merging as admin

File: checkstyle-config/pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@ limitations under the License.
     <url>https://github.com/GoogleCloudPlatform/java-repo-tools</url>
     <connection>scm:git:https://github.com/GoogleCloudPlatform/java-repo-tools.git</connection>
     <developerConnection>scm:git:ssh://git@github.com/GoogleCloudPlatform/java-repo-tools.git</developerConnection>
-    <tag>v1.0.3-alpha</tag>
+    <tag>v1.0.1</tag>
```
Comment (tswast) [inline checkstyle-config/pom.xml]: Looks like the version went backwards?

Comment (jabubake) [inline checkstyle-config/pom.xml]: I was also confused by this, and the parent pom refers to 1.0.0 for checkstyle : https://github.com/GoogleCloudPlatform/java-repo-tools/blob/master/pom.xml

Comment (lesv) [inline checkstyle-config/pom.xml]: LOL - yep - the 1.0.1 would have been the official number.  Of course, I ran into trouble uploading, on Saturday, and didn't get back to it.

**Comment by lesv** [comment]
Released & published

**Pull request merged**